src/commands/bin.rs

```diff
@@ -0,0 +1,24 @@
+use crate::config::Config;
+use crate::data::manifest::PACKAGES_DIR_NAME;
+use crate::dataflow::bin_script::BIN_DIR_NAME;
+use std::env;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+pub struct BinOpt {
+    /// Get the global .bin dir
+    #[structopt(short = "g", long = "global")]
+    pub global: bool,
+}
+
+pub fn bin(options: BinOpt) -> Result<(), failure::Error> {
```

Comment:
If the current dir is not a valid wapm project, we should raise an Error.
Instead of returning `$pwd/wapm_packages/.bin`.